src/main/kotlin/no/nav/dagpenger/regel/periode/Periode.kt

```diff
@@ -64,6 +66,17 @@ class Periode(private val env: Environment) : River() {
         )
         return props
     }
+
+    override fun onFailure(packet: Packet): Packet {
+        packet.addProblem(
+            Problem(
+                type = URI("urn:dp:error:regel"),
+                title = "Ukjent feil ved bruk av perioderegel",
+                instance = URI("urn:dp:regel:periode")
```

Comment:
minor: change `urn:dp:regel:periode` to `urn:dp:error:regel:periode`  to keep semantic?

Reply:
Hmm, kan sikkert det! Jeg tenkte at instance bare identifiserer reglen, og ikke direkte har med at det er en feil

Reply:
ah, ja - godt poeng.

Reply:
bare la det være